kadai-adapter-camunda-spring-boot-test/src/test/java/io/kadai/adapter/integration/CamundaHealthCheckTest.java

```diff
@@ -0,0 +1,166 @@
+package io.kadai.adapter.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.method;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.requestTo;
+import static org.springframework.test.web.client.response.MockRestResponseCreators.withStatus;
+import static org.springframework.test.web.client.response.MockRestResponseCreators.withSuccess;
+
+import io.kadai.adapter.impl.LastSchedulerRun;
+import io.kadai.adapter.monitoring.CamundaHealthCheck;
+import io.kadai.adapter.monitoring.SchedulerHealthCheck;
+import io.kadai.adapter.test.KadaiAdapterTestApplication;
+import io.kadai.common.test.security.JaasExtension;
+import io.kadai.common.test.security.WithAccessId;
+import java.time.Duration;
+import java.time.Instant;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.client.ExpectedCount;
+import org.springframework.test.web.client.MockRestServiceServer;
+import org.springframework.web.client.RestTemplate;
+
+@SpringBootTest(
+    classes = KadaiAdapterTestApplication.class,
+    webEnvironment = WebEnvironment.DEFINED_PORT)
+@AutoConfigureWebTestClient
+@ExtendWith(JaasExtension.class)
+@ContextConfiguration
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Yes, 
DirtiesContext is used to clean the application context after running the health check test, so that other Tests will not be disturbed by previously existing context.
TestInstance is used so that I can use the @AfterAll method which calls the setIsInitialized from the AbsIntegrationTest. Without this, I cannot call methods from super class